src/scss/includes/search_form.scss

```diff
@@ -302,8 +302,11 @@ input[type="search"] {
 
   .autocomplete_suggestions {
```

Comment:
Using this selector, also affects the main search bar suggest. It may be done later, but I think that's not what we want in this PR and issue.

Reply:
it's ok, we decided to enable fullscreen suggests everywhere within this issue